examples/mnist.py

```diff
@@ -66,7 +72,11 @@ def load_data():
 
 def build_model(input_dim, output_dim,
                 batch_size=BATCH_SIZE, num_hidden_units=NUM_HIDDEN_UNITS):
-
+    """Create a symbolic representation of a neural network with
+    `intput_dim` input nodes, `output_dim` output nodes, a training function
+    with a mini-batch size of `batch_size` and `num_hidden_units` per hidden
+    layer.
```

Comment:
+1 for documenting, but I'd change that a bit. It sounds as if the training function had a particular batch size and a particular number of hidden units now, but in fact both apply to the network, and the training function is not created in `build_model` at all.